### PR TITLE
support for typescript

### PR DESCRIPTION
**Pull request by AEPKILL**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,7 +41,7 @@
     "rollup": "^0.50.0",
     "rollup-plugin-babel": "^3.0.2",
     "sierra-library": "^2.0.0",
-    "webpack": ">= 2.2.0",
+    "webpack": ">=2.0 <4.0.0",
```
Comment (jalkoby) [inline package.json:44]: please move it into "hot module reload" PR as the current version supports webpack 4

Comment (jalkoby) [reply package.json:44]: "html-webpack-plugin" is not a dependency of sass-webpack-plugin

Comment (AEPKILL) [reply package.json:44]: I just change webpack version from `devDependencies`, because of the demo need `html-webpack-plugin`.

Comment (jalkoby) [reply package.json:44]: ahh, sorry missed that

Comment (AepKill) [reply package.json]: you can clone https://github.com/AepKill/sass-webpack-plugin.git to try it, it's work ok.

**Comment by jalkoby** [comment]
thanks 🙏

**Pull request merged**